units/include/scipp/units/neutron.h

```diff
@@ -82,7 +82,9 @@ template <>
 struct boost::units::base_unit_info<
     scipp::units::detail::tof::wavelength_base_unit> {
   static std::string name() { return "angstroms"; }
-  static std::string symbol() { return u8"\u212B"; }
+  static std::string symbol() {
+    return reinterpret_cast<const char *>(u8"\u212B");
```

Comment:
Can you elaborate why this is needed? Is there a nicer solution that does not require `reinterpret_cast`, e.g., just returning a `std::string`?

Reply:
This is due to the support for char8_t introduced in C++20. There is a paper listing possible steps to make things backward compatible here http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1423r2.html . I'll have a go at some other solutions.

Reply:
Can you add that explanation as a comment? Cheers!

Reply:
@Matthew-Andrew Can you add an explanation as a comment? And then I will merge this.